src/cucumberForgeDesktop.js

```diff
@@ -87,7 +88,7 @@ const saveOutput = () => {
 };
 
 const setNewFolderPath = (folderPath) => {
-  selectedFolderPath = folderPath;
+  selectedFolderPath = isDirectory(folderPath) ? folderPath : path.dirname(folderPath);
```

Comment:
This is kind of a weird thing that happened when I upgraded the Electron version.  The path getting passed into this function  (from the HTML input) used to just be the path to the selected directory. After the upgrade this path being passed is actually the path to the first file in the directory.  So, I just updated this function to support either.